src/utils.cairo

```diff
@@ -32,6 +32,32 @@ impl HashPartialEq of PartialEq<Hash> {
     }
 }
 
+impl HashPartialOrd of PartialOrd<Hash> {
```

Comment:
We want `PartialEq` not `PartialOrd`, right? What partial order one can have with hashes?